fjord/feedback/views.py

```diff
@@ -22,6 +22,7 @@
 )
 from fjord.feedback import config
 from fjord.feedback import models
+from fjord.feedback.models import Response
 from fjord.feedback.forms import ResponseForm
```

Comment:
Nit: We try to order imports alphabetically, so "from fjord.feedback.models..." should come after "from fjord.feedback.forms..."